ATB.BL/DataAccess/FileBookingRepository.cs

```diff
@@ -0,0 +1,119 @@
+﻿using ATB.Entities;
+using ATB.Services;
+
+namespace ATB.DataAccess
+{
+    internal class FileBookingRepository : IBookingRepository
+    {
+        private string bookingsFilePath = "files/Bookings.txt";
+        public void AddBooking(Booking booking)
+        {
+            try
+            {
+                string bookingLine = $"{booking.Passenger.PassengerId},{booking.Flight.FlightId},{booking.FClass}";
+                File.AppendAllLines(bookingsFilePath, new[] { bookingLine });
+                Console.WriteLine("Booking added successfully!");
+            }
+            catch (IOException ex)
+            {
+                Console.WriteLine($"An error occurred while adding the booking: {ex.Message}");
+            }
```

Comment:
I think this was from a previous PR, but do we want to catch this exception here?